### ISSUE TITLE
MIN_TRADE_REQUIREMENT_NOT_MET

### ISSUE THREAD

**Issue by pavlo-savchuk** [gcarq/freqtrade#213]
![default](https://user-images.githubusercontent.com/19304066/34207273-784d1654-e592-11e7-98ef-c0b59227139d.png)

Bittrex requires a minimum amount of 50k satoshi
Current price is 54k satoshi

**Comment by csplkiller** [comment]
this error does not apply to this project

**Comment by gcarq** [comment]
Yes this error does not apply to this project, but we're aware that the bittrex API acts very weird sometimes. Best thing would be to open a bittrex issue and ask them if they could provide more meaningful error messages or an error code table.
